extensions/Particle3D/PU/CCPUScriptTranslator.h

```diff
@@ -159,7 +159,7 @@ enum eDefineStaticToken
     TOKEN_EMITTER_PARTICLE_HEIGHT,
     TOKEN_EMITTER_PARTICLE_DEPTH,
     TOKEN_EMITTER_AUTO_DIRECTION,
-    TOKEN_EMITTER_FORCE_EMISISON,
+    TOKEN_EMITTER_FORCE_EMISSION,
```

Comment:
I think it is better to add 
TOKEN_EMITTER_FORCE_EMISSION = TOKEN_EMITTER_FORCE_EMISISON,

Or it will break compatibility.